drivers/hmis/app/models/hmis/role.rb

```diff
@@ -155,13 +155,6 @@ def self.permissions_with_descriptions
         category: 'Project Access',
         sub_category: 'Management',
       },
-      can_manage_inventory: { # TODO: deprecated, remove in next release in favor of can_manage_units below
```

Comment:
After this the 2 remaining occurrences of `can_manage_inventory` in the code are in `db/structures/primary.sql` and `drivers/datalab_testkit/spec/fixpoints/warehouse_app_test.sql`. These both look like generated files - is there a code gen step that I should run to get rid of it here, or is it fine to leave?

Reply:
Huh I've never noticed those files. It looks like maybe they get updated when we collapse migrations, which we do like every 6 months (latest update was https://github.com/greenriver/hmis-warehouse/pull/4583). So I wouldn't worry about that.

Reply:
the files in `db/structures/*.sql` are, as @gigxz noted, the files we generate as part of the migration collapses.  They shouldn't be a problem, but new installations will use them to bootstrap, so running those and then the subsequent migrations needs to work.  In this case, the column will probably be added and then removed.